index.html

```diff
@@ -35,5 +35,142 @@ <h1 class="bigtext1">I'm Alexandra <br> Glad to see you!</h1>
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Back End Dev</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2"> 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="bg">
+      <div id="about-section">
+          <h3 id="about-myself">About <br> Myself</h3>
+        <p class="about-text">I'm a software developer! I can help you build a product feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="about-connect">LET'S CONNECT</p>
+        <div class="about-links">
+          <ul>
+            <li><a href="http://twitter.com/Dammybest"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/damilare-ismaila-4a5a8b30/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="http://github.com/Bestbynature"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="http://facebook.com/bestbynature"><i class="fa-brands fa-facebook"></i></a></li>
+          </ul>
+        </div>
+        <div>
+          <button type="button" class="btn-1 left-margin">Get my resume</button>
+        </div>
+        <div class="categories left-margin">
+          <ul>
+            <li class="right-padding">Languages <i class="fa-sharp fa-solid fa-angle-down"></i></i></li>
+            <div class="frames">
+              <ul>
+                <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/javascript_logo_3.png" alt="Javascript logo">Javascript</button></a></li>
+                <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/html_5_logo.png" alt="html logo">HTML</button></a></li>
+                <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/css_3_logo.png" alt="CSS logo">CSS</button></a></li>
+            </ul>
+            </div>
+            <li class="right-padding">Frameworks <i class="fa-sharp fa-solid fa-angle-right"></i></i></li>
+            <hr>
+            <li class="right-padding">Skills <i class="fa-sharp fa-solid fa-angle-right"></i></i></li>
+            <hr>
+          </ul>
```

Comment:
- [ ] [OPTIONAL] Great job with the design. 💯 Let's try fixing the semantics of this `ul` for a better HTML structure. We should only use `li`s inside a `ul` in this case. Please refer to this [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) to get more insight on which elements are allowed inside a `ul`. 👍 

**Example**
```
<ul>
  <li class="right-padding">Languages <i class="fa-sharp fa-solid fa-angle-down"></i></i></li>
  <li class="frames">
    <ul>
       <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/javascript_logo_3.png" alt="Javascript logo">Javascript</button></a></li>
       <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/html_5_logo.png" alt="html logo">HTML</button></a></li>
       <li><a href="#"><button type="button" class="skills-btn"><img class="skills-logo" src="./images/css_3_logo.png" alt="CSS logo">CSS</button></a></li>
     </ul>
  </li>
  <li class="right-padding">Frameworks <i class="fa-sharp fa-solid fa-angle-right"></i></i></li>
  <li><hr></li>
  <li class="right-padding">Skills <i class="fa-sharp fa-solid fa-angle-right"></i></i></li>
  <li><hr></li>
</ul>
```

Reply:
The correction is duly noted. Thanks